moninag/moninag/urls.py

```diff
@@ -22,6 +22,7 @@
     url(r'^api/1/profile/', include('user_profile.urls')),
     url(r'^api/1/server/', include('server.urls')),
     url(r'^api/1/service/', include('service.urls')),
+    url(r'^api/1/contact/', include('contact.urls')),
```

Comment:
Maybe it's better to sort them alphabetically?
But it's up to you :slightly_smiling_face: 

`home` leave at the bottom anyway.